From 5bcd75ef87382aa13e4044b30f160291e8fa947e Mon Sep 17 00:00:00 2001
From: YaleiZang <yalei.zang@airoha.com>
Date: Mon, 8 Jun 2026 20:12:49 +0800
Subject: [PATCH] airoha: enable OF_LIBFDT_OVERLAY support for 7581/7583

Enable CONFIG_OF_LIBFDT_OVERLAY to allow loading and applying device tree overlays
at boot time. This is necessary for systems requiring flexible hardware description
via DT overlay, such as dynamic board variants or runtime configurable peripherals.

Signed-off-by: YaleiZang <yalei.zang@airoha.com>
---
 configs/an7581_evb_defconfig | 1 +
 configs/an7583_evb_defconfig | 1 +
 2 files changed, 2 insertions(+)

--- a/configs/an7581_evb_defconfig
+++ b/configs/an7581_evb_defconfig
@@ -94,3 +94,4 @@ CONFIG_DM_SPI=y
 CONFIG_AIROHA_SNFI_SPI=y
 # CONFIG_UBIFS_SILENCE_MSG is not set
 CONFIG_SHA512=y
+CONFIG_OF_LIBFDT_OVERLAY=y
--- a/configs/an7583_evb_defconfig
+++ b/configs/an7583_evb_defconfig
@@ -94,3 +94,4 @@ CONFIG_DM_SPI=y
 CONFIG_AIROHA_SNFI_SPI=y
 # CONFIG_UBIFS_SILENCE_MSG is not set
 CONFIG_SHA512=y
+CONFIG_OF_LIBFDT_OVERLAY=y
